src/System.Runtime/tests/Performance/Perf.String.netcoreapp.cs

```diff
@@ -10,15 +10,18 @@ public partial class Perf_String
 
         [Benchmark]
         [MemberData(nameof(ContainsStringComparisonArgs))]
-        public void Contains(StringComparison comparisonType, int size)
+        public void Contains(StringComparison comparisonType, int[] sizes)
```

Comment:
Is this what was intended, that the single benchmark execution runs over many different sizes, rather than having a different execution of the benchmark for each size?  I'd have thought the problem was in the results produced by Permutations, not in this actual perf test.
cc: @mellinoe